Dockerfile.multistage

```diff
@@ -0,0 +1,24 @@
+# Start from golang v1.10.4 base image
```

Comment:
That comments looks like how to tutorial comments. But in fact they are absolutely useless and doesn't give any information. Write comments only in difficult, not obvious places. It will keep you code clean.